src/ros_filter.cpp

```diff
@@ -2965,7 +2968,7 @@ bool RosFilter<T>::preparePose(
   tf2::Transform target_frame_trans;
   bool can_transform = ros_filter_utilities::lookupTransformSafe(
     tf_buffer_.get(), final_target_frame, pose_tmp.frame_id_,
-    rclcpp::Time(tf2::timeToSec(pose_tmp.stamp_)), tf_timeout_,
+    rclcpp::Time(tf2::timeToSec(pose_tmp.stamp_), RCL_ROS_TIME), tf_timeout_,
```

Comment:
Shouldn't we use the `node->get_clock()` type of time? This assumes simulation time.